docs/examples/101/data-factory-v2/main.bicep

```diff
@@ -0,0 +1,14 @@
+param name string = 'myv2datafactory'
+
+param location string {
+  default: resourceGroup().location
+}
+
+resource dataFactory 'Microsoft.DataFactory/factories@2018-06-01' = {
+  name: name
+  location: location
+  identity: {
+    type: 'SystemAssigned'
+  }
+  properties: {}
```

Comment:
assuming you don't get a warning, this line should be removed

Reply:
I get a warning on the ARM template generated.

Reply:
interesting - can you show me the warning you are getting? I am not seeing it with ARM Tools and the `2019-04-01` root schema

Reply:
![Capture d’écran 2020-11-30 à 17 06 01](https://user-images.githubusercontent.com/5141735/100633418-5e632480-332e-11eb-92c0-6fea99302b34.png)

Reply:
Also the 2019-04-01 schema is not available with autocompletion.

Reply:
Thanks - I will create a separate issue to track this discrepancy between Bicep and ARM tools